specs/commandline/certificate.md

```diff
@@ -151,31 +151,31 @@ Upon successful adding, the certificate files are added into directory`{NOTATION
 notation certificate list
 ```
 
-Upon successful listing, all the certificate files in the trust store are printed out in a format of absolute filepath. If the listing fails, an error message is printed out with specific reasons. Nothing is printed out if the trust store is empty.
+Upon successful listing, all the certificate files in the trust store are printed out with information of store type, store name and certificate file name. If the listing fails, an error message is printed out with specific reasons. Nothing is printed out if the trust store is empty.
```

Comment:
Is it possible to show an example of output, so that we can review the name of the column in a table view

Reply:
Added example.